### ISSUE TITLE
Adhere to ipywidget standards in trait names.

### ISSUE THREAD

**Issue by yakutovicha** [aiidalab/aiidalab-widgets-base#72]
As [proposed](https://github.com/aiidalab/aiidalab-widgets-base/pull/69#issuecomment-610366361) by @csadorf:

We should consider renaming the primary "value" of a widget, actually `value`, since the type is already specified through the class name. For example, the "value" of an `IntSlider` is not `number` or `integer_number`, but `value`. 

In the case of computers it would look as follows:
 - `selected_computer` -> `value`
 - `computers` -> `options`

I think we should go through the deprecation mechanism here, to be decided.

**Comment by csadorf** [comment]
> I think we should go through the deprecation mechanism here, to be decided.

I don't think this is a priority right now, so we can hold off for now, but either way if we decide to do this, at the current development stage, there is no need for a deprecation period, we can just apply these changes.

**Comment by danielhollas** [comment]
I'll move this out of the v3 milestone as I don't think it is worth the churn right now. Let me know if you disagree @yakutovicha @edan-bainglass
